client/state/data-layer/wpcom/sites/rewind/schema.js

```diff
@@ -0,0 +1,102 @@
+/** @format */
+
+export const credential = {
+	type: 'object',
+	properties: {
+		still_valid: { type: 'boolean' },
+		type: { type: 'string', enum: [ 'auto', 'ftp', 'sftp', 'ssh' ] },
+		host: { type: 'string' },
+		port: { type: 'integer' },
+	},
+	required: [ 'still_valid', 'type' ],
+};
+
+export const download = {
+	type: 'object',
+	properties: {
+		downloadId: { type: 'integer' },
+		rewindId: { type: 'string' },
+		backupPoint: { type: 'integer' },
+		startedAt: { type: 'integer' },
+		downloadCount: { type: 'integer' },
+		validUntil: { type: 'integer' },
+	},
+	required: [ 'downloadId', 'rewindId', 'backupPoint', 'startedAt' ],
+};
+
+export const restore = {
+	type: 'object',
+	properties: {
+		status: { type: 'string', enum: [ 'queued', 'running' ] },
```

Comment:
should we also have `inactive` here?

Reply:
thanks! I wasn't sure which states existed